interfaces/builtin/i2c.go

```diff
@@ -42,13 +42,19 @@ const i2cBaseDeclarationSlots = `
     deny-auto-connection: true
 `
 
-const i2cConnectedPlugAppArmor = `
+const i2cConnectedPlugAppArmorPath = `
 # Description: Can access I2C controller
 
 %s rw,
 /sys/devices/platform/{*,**.i2c}/%s/** rw,
 `
 
+const i2cConnectedPlugAppArmorSysfsName = `
+# Description: Can access I2C sysfs name
+
+/sys/bus/i2c/devices/%s/** rw,
```

Comment:
On my system 18.04 classic system, everything in `/sys/bus/i2c/devices/*` is a symlink to `/sys/devices/pci[0-9a-f:]+/[0-9a-f:]+/*`. Is `/sys/bus/i2c/devices` confirmed as the correct path when using sysfs name?

Reply:
I see the same. It sounds like we should check in the code if its a symlink and resolve it before writing the apparmor profile. Does that sounds reasonable @jdstrand ?